tests/unit/plugins/modules/kubevirt_fixtures.py

```diff
@@ -66,8 +65,8 @@ def base_fixture(monkeypatch):
     openshift.dynamic.Resource.search = MagicMock()
     openshift.dynamic.Resource.watch = MagicMock()
```

Comment:
Again, do we still need openshift client?

Reply:
See above, thanks for the review, much appreciated.